src/alto-ui/Datagrid/components/DatagridGroupedRow/DatagridGroupedRow.js

```diff
@@ -41,6 +41,9 @@ const DatagridGroupedRow = ({
   ...datagridRowProps
 }) => {
   const { id, rowKeyField, groupedByColumnKey, groupedSummaryColumnKeys, labels } = context;
+  const labelValue = Object.keys(firstRowInGroup).includes(`${groupedByColumnKey}_lbl`)
+    ? firstRowInGroup[`${groupedByColumnKey}_lbl`]
```

Comment:
Maybe we need to have this logic in a function `getLabelValue` on webApp and pass that function to alto `Datagrid` because it's really linked to WebApp implementation

@renaudtertrais what do you think?